riemann-java-client/src/main/java/com/aphyr/riemann/client/TcpHandler.java

```diff
@@ -1,26 +1,9 @@
 package com.aphyr.riemann.client;
 
-import java.net.InetSocketAddress;
```

Comment:
There's a lot of import deletions here--I'm guessing your IDE cleaned up the imports automatically? Can you confirm everything still builds OK? :)